src/Microsoft.Azure.SignalR.Management/HubInstanceFactories/MultiEndpointConnectionContainerFactory.cs

```diff
@@ -27,31 +24,16 @@ public MultiEndpointConnectionContainerFactory(IServiceConnectionFactory connect
             _router = router;
         }
 
-        public MultiEndpointServiceConnectionContainer GetOrCreate(string hubName, ILoggerFactory loggerFactoryPerHub = null)
+        public MultiEndpointServiceConnectionContainer CreateAndStart(string hubName)
         {
-            return _hubConnectionContainers.GetOrAdd(hubName, (hubName) => Create(hubName, loggerFactoryPerHub)).Value;
```

Comment:
why _hubConnectionContainers should be removed?

Reply:
We don't need to reuse the container now.